src/Sylius/Behat/Context/Ui/Admin/ManagingOrdersContext.php

```diff
@@ -248,11 +257,12 @@ public function iShouldNotBeAbleToResendTheShipmentConfirmationEmail(): void
     }
 
     /**
+     * @Then I should see :amount orders in the list
```

Comment:
As it is not consistent with the argument
```suggestion
     * @Then I should see :count orders in the list
```